WcaOnRails/app/helpers/competitions_helper.rb

```diff
@@ -13,4 +13,38 @@ def competition_message_for_user(competition, user)
                 end
     messages.join(' ')
   end
+
+  def announced_class(distance_announced)
+    if distance_announced <= 21
+      "alert-danger"
+    elsif distance_announced <= 28
+      "alert-orange"
+    else
+      "alert-green"
+    end
+  end
+
+  def report_class(distance_report)
+    if distance_report > 21
+      "alert-danger"
+    elsif distance_report > 14
+      "alert-orange"
+    elsif distance_report > 7
+      "alert-success"
+    else
+      "alert-green"
+    end
+  end
+
+  def results_class(distance_results)
+    if distance_results > 21
```

Comment:
I'm not in love with the numbers 21, 14, 7 being repeated throughout this file, did you consider pulling them into constants?